diff --git a/hw/i386/pc_q35.c b/hw/i386/pc_q35.c
index 01c56bf55e..c913432cdc 100644
--- a/hw/i386/pc_q35.c
+++ b/hw/i386/pc_q35.c
@@ -339,7 +339,7 @@ static void pc_q35_machine_options(MachineClass *m)
     pcmc->default_cpu_version = 1;
 
     m->family = "pc_x570";
-    m->desc = "AMD Ryzen 7 7700X 8-Core Processor";
+    m->desc = "AMD Ryzen 7 7600X 6-Core Processor";
     m->units_per_default_bus = 1;
     m->default_machine_opts = "firmware=bios-256k.bin";
     m->default_display = "std";